.gitignore

```diff
@@ -10,4 +10,7 @@ bin
 integration-tests/cache/
 integration-tests/artifacts/
 **/*.env
-.devnet
\ No newline at end of file
+.devnet
+
+# Ignore local fuzzing results
+**/testdata/fuzz/
```

Comment:
If I understand correctly, failures are added to the corpus which then seed future tests and they are held at this directory. Wouldn't we want to keep those seeds?

Reply:
Maybe, I'm not so sure if we want it here in the source-code, or in a separate repo focused on fuzzing maybe. cc @trianglesphere do you want to check these files into git, if there are any?

Reply:
I don't think they should be added to git. It'd probably be good practice to add them with `f.Add(...)` at the start.
The long term solution is a fuzzing cluster.